Formula/procps.rb

```diff
@@ -0,0 +1,34 @@
+class Procps < Formula
+  desc "Utilities for browsing procfs"
+  homepage "https://gitlab.com/procps-ng/procps"
+  url "https://gitlab.com/procps-ng/procps/-/archive/v3.3.16/procps-v3.3.16.tar.gz"
+  sha256 "7f09945e73beac5b12e163a7ee4cae98bcdd9a505163b6a060756f462907ebbc"
+  license "GPL-2.0-or-later"
+  head "https://gitlab.com/procps-ng/procps.git"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "gettext" => :build
+  depends_on "libtool" => :build
+  depends_on "pkg-config" => :build
+  depends_on :linux
```

Comment:
Are you sure this is Linux-only? Had a quick skim of their repo and their code looks like it could be portable...

Reply:
According to here: https://trac.macports.org/ticket/32111 it is not, because it assumes the existence of `/proc`, which does not exist on macOS.